login-workflow/docs/screens/site-options.md

```diff
@@ -0,0 +1,31 @@
+# SiteOptionsScreen
+
+A component to use for DERMS Auth workflow. It renders a a message, an icon, and a two buttons with `Create a New Organization` and `Join Existing Organization` buttons.
```

Comment:
Typo 
`It renders a a message, an icon, and a two buttons`

There are 2 `a`s before `message`.
Adding `a` before `two` is grammatically wrong.